js/apps/admin-ui/src/user/UserProfileFields.tsx

```diff
@@ -221,3 +224,19 @@ function determineInputType(
 
 const isValidInputType = (value: unknown): value is InputType =>
   typeof value === "string" && value in FIELDS;
+
+const attributeGroupLabel = (
+  group: UserProfileAttributeGroupMetadata,
+  t: TFunction,
+) =>
+  (isBundleKey(group.displayHeader)
+    ? t(unWrap(group.displayHeader!))
+    : group.displayHeader) || group.name;
+
+const attributeGroupDescriptionLabel = (
+  group: UserProfileAttributeGroupMetadata,
+  t: TFunction,
+) =>
+  isBundleKey(group.displayDescription)
+    ? t(unWrap(group.displayDescription!))
+    : group.displayDescription;
```

Comment:
This code looks the same between these functions, aside from the field they operate on. Perhaps this can be done generically? I am also seeing a [similar function](https://github.com/keycloak/keycloak/blob/fe0a9459dd5ae4f400abb17c6b48184a37e98293/js/apps/account-ui/src/personal-info/utils.ts#L9-L12) in the utils that can get the same treatment.